packages/core/src/utils/cursor-pagination/helpers.ts

```diff
@@ -55,13 +57,11 @@ export async function singleColPaginatedQuery<DB, TB extends keyof DB, T>(
 
   // Take N+1 to determine if there is a record behind `take`.
   const pageWithNextLength = take + 1
-  let pagedQuery = query
-
-  pagedQuery = query.limit(pageWithNextLength)
-  pagedQuery = pagedQuery.orderBy(sql.raw(`${options.column} ${order}`))
+  let pagedQuery = query.limit(pageWithNextLength)
+  pagedQuery = pagedQuery.orderBy(options.column, order)
   if (cursor !== undefined) {
     const decodedCursor = decodeCursor(cursor)
-    pagedQuery = pagedQuery.where(sql.raw(`${options.column} <= '${decodedCursor}'`))
+    pagedQuery = pagedQuery.where(options.column, order === "asc" ? ">" : "<", decodedCursor)
```

Comment:
Right now the `getNextCursor` function assigns the next cursor to the take+1 record. So it should be >= and <= right? Or else we skip the take+1 record?

Reply:
Yes, fixed it in https://github.com/dotkom/monoweb/pull/926/commits/a9bc2e0ddd4fa1814a13ec138cd0fe5c1e5b8690